app/lib/activitypub/dereferencer.rb

```diff
@@ -44,7 +44,7 @@ def perform_request(uri, headers: nil)
 
     req = Request.new(:get, uri)
 
-    req.add_headers('Accept' => 'application/activity+json, application/ld+json')
+    req.add_headers('Accept' => 'application/ld+json; profile="https://www.w3.org/ns/activitystreams", application/activity+json, application/ld+json')
```

Comment:
this weirdo special case fetch code was written in 2020 and has remained pretty much unchanged since.... I'm okay with updating this but we should remove the `application/ld+json` type to avoid confusion and in general we should consider if we can remove this special case codepath and fold this into the normal Fetch* services

Reply:
No rationale has been provided for the `application/ld+json` case. I'd indeed just use the full profile here.

Reply:
Done; only JSON-LD with the AS profile is used now